src/parser/core/modules/Swiftcast.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+
+import {BuffWindowModule} from 'parser/core/modules/BuffWindow'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+import {getDataBy} from 'data/getDataBy'
+
+export default class Swiftcast extends BuffWindowModule {
+	static handle = 'swiftcast'
+	static title = t('core.swiftcast.title')`Swiftcasts`
```

Comment:
Might be more appropriate to call this `Swiftcast` or `Swiftcast Usage` to better match the other modules featured for other jobs.

Reply:
That's definitely more accurate. I'll get this fix up as part of my next commit.